src/blocks/shape-divider/deprecated.js

```diff
@@ -199,7 +200,7 @@ const deprecated = [
 			...metadata.attributes,
 			shapeHeight: { type: 'number', default: 100 },
 			backgroundHeight: { type: 'number', default: 50 },
-			[ attr ]: false,
+			[ attr ]: { type: 'boolean', default: false },
```

Comment:
Deprecation Jest tests were failing on this attribute because the block parser was choking while checking for the `default` property. Now that a definition is an actual object, the parser works as expected.